android/app/src/main/java/org/fossasia/openevent/data/Session.kt

```diff
@@ -0,0 +1,87 @@
+package org.fossasia.openevent.data
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.databind.PropertyNamingStrategy
+import com.fasterxml.jackson.databind.annotation.JsonNaming
+import com.github.jasminb.jsonapi.IntegerIdHandler
+import com.github.jasminb.jsonapi.annotations.Id
+import com.github.jasminb.jsonapi.annotations.Relationship
+import com.github.jasminb.jsonapi.annotations.Type
+
+import io.realm.RealmList
+import io.realm.RealmObject
+import io.realm.annotations.Index
+import io.realm.annotations.PrimaryKey
+
+@Type("session")
+@JsonNaming(PropertyNamingStrategy.KebabCaseStrategy::class)
+open class Session(
+        @PrimaryKey
+        @Id(IntegerIdHandler::class)
+        var id: Int? = null,
+        @Index
+        var title: String? = null,
+        var subtitle: String? = null,
+        var shortAbstract: String? = null,
+        var longAbstract: String? = null,
+        var comments: String? = null,
+        var startsAt: String? = null,
+        var endsAt: String? = null,
+        var language: String? = null,
+        var slidesUrl: String? = null,
+        var videoUrl: String? = null,
+        var audioUrl: String? = null,
+        var signupUrl: String? = null,
+        var state: String? = null,
+        var level: String? = null,
+        @Index
+        var startDate: String? = null,
+        var isBookmarked: Boolean = false,
+        var createdAt: String? = null,
+        var deletedAt: String? = null,
+        var submittedAt: String? = null,
+        @Relationship("is-mail-sent")
+        @field:JsonProperty("is-mail-sent")
+        var isMailSent: Boolean = false,
+        @Relationship("session-type")
+        var sessionType: SessionType? = null,
+        @Relationship("track")
+        var track: Track? = null,
+        @Relationship("microlocation")
+        var microlocation: Microlocation? = null,
+        @Relationship("speakers")
+        var speakers: RealmList<Speaker>? = null
+) : RealmObject() {
+
+    companion object {
+        /* Sort criteria */
+        @JvmStatic
+        val TITLE = "title"
+        // Track is object in Realm. So sort by track.name
+        @JvmStatic
+        val TRACK = "track.name"
+        @JvmStatic
+        val START_TIME = "startsAt"
+    }
```

Comment:
If `@JvmField` is used instead of `@JvmStatic`. then we will be able to access the properties as we were in Java

Reply:
This is not required as we converted this back to java and exported it into a new module